administrator/language/de-DE/com_joomlaupdate.ini

```diff
@@ -74,38 +93,39 @@ COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_NONE="Es sind keine Erweiterungen insta
+COM_JOOMLAUPDATE_VIEW_DEFAULT_NON_CORE_PLUGIN_CONFIRMATION="I accept the warnings about potentially incompatible extensions and wish to proceed with the update."
 COM_JOOMLAUPDATE_VIEW_DEFAULT_NOUPDATES="Keine Updates verfügbar"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_NOUPDATESNOTICE="Es ist bereits die aktuellste Joomla-Version %s installiert."
 COM_JOOMLAUPDATE_VIEW_DEFAULT_PACKAGE="Update-Paket-URL"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_PACKAGE_INFO="You can also download <a href=\"%s\" class=\"alert-link\" target=\"_blank\" rel=\"noopener noreferrer\">the update package</a> and install it manually."
 COM_JOOMLAUPDATE_VIEW_DEFAULT_PACKAGE_REINSTALL="Neuinstallations-Paket-URL"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_PHP_VERSION_NOT_SUPPORTED="PHP-Version wird nicht unterstützt"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_PHP_VERSION_NOT_SUPPORTED_DESC="Ein Update für Joomla! %1$s wurde gefunden, allerdings entspricht die aktuell installierte PHP-Version nicht mehr <a href='https://downloads.joomla.org/technical-requirements' target='_blank' rel='noopener noreferrer'>den Mindestanforderungen für Joomla %1$s</a> oder wird nicht mehr unterstützt."
-COM_JOOMLAUPDATE_VIEW_DEFAULT_POTENTIALLY_DANGEROUS_PLUGIN="Ein möglicherweise schwerwiegendes Problem."
-COM_JOOMLAUPDATE_VIEW_DEFAULT_POTENTIALLY_DANGEROUS_PLUGIN_CONFIRM_MESSAGE="Die Warnungen über potenziell inkompatible Plugins <strong>wirklich ignorieren</strong> und mit dem Upgrade fortfahren?"
-COM_JOOMLAUPDATE_VIEW_DEFAULT_POTENTIALLY_DANGEROUS_PLUGIN_DESC="Diese Erweiterung enthält ein Plugin, das zu einer fehlerhaften Aktualisierung führen kann.<br><br>Um das Joomla-Upgrade sicher durchzuführen, sollte diese Erweiterung entweder auf eine Version aktualisiert werden, die mit der Zielversion von Joomla kompatibel ist oder das/die betroffene(n) Plugin(s) deaktivieren und erneut überprüfen.<br><br>Weitere Informationen zu den betroffenen Plugins sind im Tab „Live-Update“ zu finden."
-COM_JOOMLAUPDATE_VIEW_DEFAULT_POTENTIALLY_DANGEROUS_PLUGIN_LIST="Die folgenden Plugins könnten beim Upgrade Probleme verursachen"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_POTENTIALLY_DANGEROUS_PLUGIN="Ein mögliches Problem."
+COM_JOOMLAUPDATE_VIEW_DEFAULT_POTENTIALLY_DANGEROUS_PLUGIN_CONFIRM_MESSAGE="Die Warnungen über potenziell inkompatible Erweiterungen <strong>wirklich ignorieren</strong> und mit dem Update fortfahren?"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_POTENTIALLY_DANGEROUS_PLUGIN_DESC="Diese Erweiterung enthält ein Plugin, das zu einer fehlerhaften Aktualisierung führen kann.<br><br>Um das Joomla-Update sicher durchzuführen, sollte diese Erweiterung entweder auf eine Version aktualisiert werden, die mit der Zielversion von Joomla kompatibel ist oder das/die betroffene(n) Plugin(s) deaktivieren und erneut überprüfen.<br><br>Weitere Informationen zu den betroffenen Plugins sind im Tab „Live-Update“ zu finden."
```

Comment:
```suggestion
COM_JOOMLAUPDATE_VIEW_DEFAULT_POTENTIALLY_DANGEROUS_PLUGIN_DESC="Diese Erweiterung enthält ein Plugin, das zu einer fehlerhaften Aktualisierung führen kann.<br><br>Um das Joomla-Update sicher durchzuführen, sollte diese Erweiterung entweder auf eine Version aktualisiert werden, die mit der Zielversion von Joomla kompatibel ist oder das/die betroffene(n) Plugin(s) deaktivieren und erneut überprüfen."
```

Reply:
den tab gibts in 4.x nicht mehr muss / wird auch noch upstream gefixt werden müssen